PR_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Motivation/Description of the PR
+
+Applicable helpers:
+
+[ ] Webdriver
+[ ] Puppeteer
+[ ] Nightmare
+[ ] REST
+[ ] Appium
+[ ] Protractor
+[ ] TestCafe
+
+- Description of this PR, which problem it solves
+- A link to the corresponding issue (if applicable).
+
+## Type of change
+
+- [ ] Breaking changes
+- [ ] New functionnality
+- [ ] Bug fix
+- [ ] Documentation change/update
+- [ ] Hot fix
+- [ ] Markdown files fix - not related to source code
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] Tests have been added
+- [ ] Documentation has been added (Run `robo docs`)
+- [ ] Lint checking (Run `npm run lint`)
```

Comment:
Lint and the test running on each push and in the pipeline, why is this added to checklist?

Reply:
with some PRs, I still see the lint issues, so better to add this so they could fix it before committing.